Library/Formula/htop.rb

```diff
@@ -0,0 +1,31 @@
+class Htop < Formula
+  desc "Improved top (interactive process viewer)"
+  homepage "https://github.com/hishamhm/htop"
+  url "https://github.com/hishamhm/htop/archive/2.0.0.tar.gz"
+  sha256 "2522a93792dfee188bfaff23f30332d1173460c95f9869588398e9bdd3a0491b"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  conflicts_with "htop-osx", :because => "both install an `htop` binary"
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
It's actually more important for homebrew, since homebrew doesn't require root. For other OSes, where a package manager requires something like `sudo apt-get install htop`, you can assume that the user who installs this package has root access and therefore can run htop as root.